VMEncryption/main/DiskUtil.py

```diff
@@ -735,6 +735,8 @@ def query_dev_sdx_path_by_uuid(self, uuid):
     def query_dev_id_path_by_sdx_path(self, sdx_path):
         """
         return /dev/disk/by-id that maps to the sdx_path, otherwise return the original path
+        Update: now we have realised that by-id is not a good way to refer to devices (they can change on reallocations or resizes).
+        Try not to use this- use get_stable_path_from_sdx instead
```

Comment:
Add a warning log statement. If this gets used, we would know.

Reply:
So far, there are no references to this method outside of an OS Encryption code path.
And the by-id path returned there is not added to azure_crypt_mount and doesn't contribute to the problem.